.github/workflows/schedule.yml

```
@@ -4,6 +4,9 @@ on:
 
 name: Update and release
 
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
 jobs:
   updateDataAndRelease:
     name: Update data and release
```

According to your description, would it be better to change to the following form?

```suggestion
permissions: {}

jobs:
  updateDataAndRelease:
    permissions:
      contents: read # to fetch code (actions/checkout)

    name: Update data and release
```